.gitignore

```diff
@@ -6,3 +6,4 @@ bin/configlet
 bin/configlet*
 deps
 _build
+.elixir_ls/
```

Comment:
so to not include ElixirLS build files from VSCode

Reply:
In my opinion this should **not** be part of a per-project `.gitignore` but the users global one instead, see also https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
Sure, can do that